django/template/base.py

```diff
@@ -682,22 +714,19 @@ def __init__(self, token, parser):
         self.filters = filters
         self.var = var_obj
 
-    def resolve(self, context, ignore_failures=False):
+    def resolve(self, context):
```

Comment:
I was assuming this wasn't necessary since `FilterExpression` isn't part of the public API, and all the internal usages have been updated.